src/resources/elements/primeDesignSystem/_styles.scss

```diff
@@ -71,7 +71,7 @@
 
   .feedback-icon {
     right: $spacing-normal;
-    top: 35%;
+    top: calc(50% - 8px); // centers the icon
```

Comment:
what is the purpose of  `.pInput-parent`?  Why is it in styles.sscss?

Reply:
The purpose of this class is to add it on the input components we create (`pinput-numeric`, `pinput-text` etc) which allows us to center the validation icons and color them. 

You may say that we can style the validation icons using the `.pInput` class but this is not possible in the case of the `pinput-numeric` because the actual input is inside another component caller `numeric-input`.

I'm open to suggestions  to move this inside the `.pInput` class 😄  @dkent600  @hiaux0 @giladt  @paweljanicki

Reply:
Not too familiar with structure of scss right now, but `.pInput-parent` in `_styles.scss` does seem too specific.
What's stopping us from creating sth like "common-input.scss" (read: shared or common place to put input stuff)

Reply:
the `.pInput-parent` class is used on the `template` tag like this: `<template class="pInput-parent ${ validationState ? 'pInput-parent-' + validationState : '' }">` (check `pinput-numeric`, `pinput-text` etc). This means that many input components will use it and that's why I added this class in `_styles.scss`. 

Nothing stops us creating `common-input.scss` tho'. Not sure how necessary is to create a component just to scope some SCSS. Your call @hiaux0 @dkent600 . Should I do it?

Reply:
I like putting it in pinput.scss.  Happy to brainstorm on how to do it.

Reply:
I moved the input related styles into pinput.scss. What do you think?